src/librustc/util/common.rs

```diff
@@ -61,7 +60,7 @@ pub enum ProfileQueriesMsg {
     /// end a task
     TaskEnd,
     /// begin a new query
-    QueryBegin(Span, QueryMsg),
+    QueryBegin(QueryMsg),
```

Comment:
`Span` is not `Send`/`Sync` if it uses thread-local interner and profile queries are sent to other threads, so I had to temporary remove spans from them.
I'll restore this back later, they will have to use `SpanData` instead of `Span` (I hoped to keep it private, but it looks like there's no better choice :( ).